CDP4Grapher/ViewModels/DiagramControlContextMenuViewModel.cs

```diff
@@ -156,16 +199,32 @@ public bool CanExportDiagram
         /// </summary>
         public DiagramControlContextMenuViewModel()
         {
+            this.UpdateProperties();
             this.InitializeCommands();
             this.CreateContextMenu();
         }
 
         /// <summary>
-        /// Initialize the <see cref="ReactiveCommand"/>
+        /// Updates all properties of this view model that need to have an initial value
         /// </summary>
-        private void InitializeCommands()
+        public void UpdateProperties()
```

Comment:
I think the name of this method should be InitializeProperties instead of UpdateProperties. 
UpdateProperties is often used as a default method to be called when data is loaded and also reloaded.